cpp/src/parquet/encryption/internal_file_decryptor.cc

```diff
@@ -148,9 +150,9 @@ InternalFileDecryptor::GetColumnDecryptorFactory(
   // The column is encrypted with its own key
   const std::string& column_key_metadata = crypto_metadata->key_metadata();
   const std::string column_path = crypto_metadata->path_in_schema()->ToDotString();
-  std::string column_key = GetColumnKey(column_path, column_key_metadata);
+  const SecureString& column_key = GetColumnKey(column_path, column_key_metadata);
 
-  return [this, aad, metadata, column_key = std::move(column_key)]() {
+  return [this, aad, metadata, column_key = column_key]() {
```

Comment:
Hmm, we must capture the column key by value here. It's not obvious that it's what `column_key = column_key` does.

Reply:
Any action item for me? Shall I leave a comment in the code on this?